Library/Formula/parse-cli.rb

```diff
@@ -0,0 +1,17 @@
+class ParseCli < Formula
+  homepage "https://parse.com"
+  url "https://www.parse.com/downloads/cloud_code/parse",
```

Comment:
The problem is that there's no guarantee of the formula being updated. Those who submitted it could lose interest or forget, maintainers could be short on time because of life, family, jobs, etc, and other contributors may not understand the tool well enough to update regularly.

In the event that the tarball changes, and consequently breaks in ` brew ` underneath users, there's no absolutely instant fix, and it could leave the formula essentially broken for a short period, which isn't a great precedent and tends to annoy users. Generally, Homebrew has been highly reluctant to accept dynamic links due to the problems caused.